plugins/modules/efs_tag.py

```diff
@@ -0,0 +1,174 @@
+#!/usr/bin/python
+"""
+Copyright: (c) 2021, Milan Zink <zeten30@gmail.com>
+GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+"""
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+DOCUMENTATION = r'''
+---
+module: efs_tag
+version_added: 1.6.0
```

Comment:
```suggestion
version_added: 2.0.0
```

The next release will be 2.0.0